Examples/SAM/Makefile

```diff
@@ -0,0 +1,52 @@
+build:
```

Comment:
this is nice, but is it required?

Reply:
The makefile was mostly intended for me to keep a list of commands at hand.  I removed it